app.js

```diff
@@ -0,0 +1,67 @@
+const initialBooks = [
+  {
+    id: 0,
+    title: 'book1',
+    author: 'author1',
+  },
+  {
+    id: 1,
+    title: 'book2',
+    author: 'author2',
+  },
+];
+
+const booksListEl = document.getElementById('books-list');
+if (localStorage.getItem('books') === null) {
+  localStorage.setItem('books', JSON.stringify(initialBooks));
+}
+
+let booksArr = JSON.parse(localStorage.getItem('books'));
+
+const remove = (removedBook) => {
+  booksArr = booksArr.filter((book) => book.id !== removedBook.id);
+  localStorage.setItem('books', JSON.stringify(booksArr));
+  booksListEl.removeChild(removedBook);
+};
```

Comment:
- [ ] The books are not being correctly removed from the local storage, it is only removed from the page. Kindly fix this. As you can see I removed the book titled "my own book" yet it exists in the local storage, and when I refresh the page it comes back. Kindly fix that.

  ![idod](https://user-images.githubusercontent.com/81468675/205080916-0145dd42-86e8-40f2-812a-0e5e7b76f915.jpg)